src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,13 @@
 package mate.academy.lib;
 
+import static java.lang.annotation.ElementType.TYPE;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(value = {TYPE})
```

Comment:
```suggestion
@Target(ElementType.TYPE)
```
I think it's more readable